appyx-components/stable/spotlight/common/src/commonMain/kotlin/com/bumble/appyx/components/spotlight/ui/sliderscale/SpotlightSliderScale.kt

```diff
@@ -18,13 +18,17 @@ import com.bumble.appyx.transitionmodel.BaseVisualisation
 
 class SpotlightSliderScale<InteractionTarget : Any>(
     uiContext: UiContext,
+    initialState: State<InteractionTarget>,
```

Comment:
Do we need the whole state or just the index?

Reply:
For now, just the index should be fine, was just following the same approach as `SpotlightHeroDefaultVisualisation`